cnf-tests/testsuites/pkg/features/features.go

```diff
@@ -0,0 +1,222 @@
+package features
+
+import (
+	"fmt"
+
+	sriovClean "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/clean"
+	sriovNamespaces "github.com/k8snetworkplumbingwg/sriov-network-operator/test/util/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/fec"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/ptp"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/sctp"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/security"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/sro"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/e2esuite/vrf"
+	testclient "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/discovery"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/namespaces"
+	testutils "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/utils"
+	numaserialconf "github.com/openshift-kni/numaresources-operator/test/e2e/serial/config"
+	perfUtils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
+	perfClean "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/clean"
+	ptpClean "github.com/openshift/ptp-operator/test/utils/clean"
+)
+
+var (
+	performanceNS    = perfUtils.NamespaceTesting
+	dpdkNS           = namespaces.DpdkTest
+	sroNS            = namespaces.SroTestNamespace
+	gatekeeperNS     = testutils.GatekeeperTestingNamespace
+	sctpNS           = sctp.TestNamespace
+	vrfNS            = vrf.TestNamespace
+	sriovNS          = sriovNamespaces.Test
+	xtu32NS          = namespaces.XTU32Test
+	ovsqosNS         = namespaces.OVSQOSTest
+	tuningcniNS      = security.TestNamespace
+	tuningcniSriovNS = security.SriovTestNamespace
+	bondcniNS        = namespaces.BondTestNamespace
+)
+
+type ConfigManager interface {
+	Setup() error
+	Cleanup() error
+}
+
+type PerformanceManager struct {
```

Comment:
Sorry I stress on naming but...
PerformanceManager is not managing Performance , DpdkManager is not managing dpdk and so on. They are not doing what their name promises, they just do setup / teardown. 

That's why I suggested the name Fixture
See for example https://en.wikipedia.org/wiki/Test_fixture or https://code-craftsmanship-saturdays.gitbook.io/software-testing-fundamentals/test-fixtures 

If you find another name that suggests setting up / tearing down I am either fine, but I think manager is not the right term here.

Same for featureManagerMap above.